app/views/layout.erb

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <%= erb :header %>
+  <%= erb :jumbotron %>
+  <div class='container'>
+    <%= erb :flash %>
+
+    <div class="row">
+      <div class="col-md-9">
+        <%= yield %>
+        <%= erb :peeps %>
+      </div>
+
+        <div class="col-md-3">
+          <% if session[:name] %>
```

Comment:
I would just have `session[:user]`, and access the name with `.name`

Even better, as part of the `SessionHelper` you made, have a `current_user` method that grabs the current user -> then we can write

```erb
<% if current_user %>
....
<% else %>
....
<% end %>
```